cassandra/pom.xml

```diff
@@ -0,0 +1,193 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>hibernate-ogm-parent</artifactId>
+        <groupId>org.hibernate.ogm</groupId>
+        <version>4.2.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+
+    <artifactId>hibernate-ogm-cassandra</artifactId>
+
+    <name>Hibernate OGM for Cassandra</name>
+    <description>Persist objects in Cassandra</description>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
+                        <cassandra_HOSTNAME>${env.cassandra_HOSTNAME}</cassandra_HOSTNAME>
```

Comment:
Setting this to another value than localhost seems to take no effect (I am running Cassandra in a VM with another IP). Probably this needs to be pushed into a system variable similar to e.g. done in `MongoDBTestHelper`. Could you also make it all upper-case to be consistent with the env variables for the other backends we have? Thx!